app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -291,20 +291,8 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
             }
             else -> {
                 context!!.webRenderComponents.sessionManager.remove()
-
-                // We can get into this state in two situations:
-                // - (1) The user is on the "home page" and the overlay is visible. In this situation we just want to
-                //       let the activity handle the back press (and leave the app).
-                // - (2) The overlay is not visible and we are on a website and can't go back further. In this case
-                //       "back" should show the overlay (the next back press closes the app). In this situation we
-                //       cannot look at the URL. We just removed the session and the new session may not be loaded yet.
-                if (!browserOverlay.isVisible) {
-                    // If the browser overlay is not visible then we show it now immediately. Depending on the loading
-                    // state of the page we m
-                    setOverlayVisible(true)
-                } else {
-                    return false
-                }
+                context!!.serviceLocator.webViewCache.doNotPersist()
```

Comment:
Talking to severin, he said that it isn't necessary to clear the WebViewCache on exit - there's one more commit on 3.1.3 that removes this line to fix #1589.
https://github.com/mozilla-mobile/firefox-tv/pull/1600/commits/484dfaaea81c5f909811de7e0c7f42652d23f3d4

Reply:
I believe I already removed that line while trying to fix a merge conflict. Should I still cherry-pick the commit?